src/plugins/shared-variables/drawing/use-variable-dialog.tsx

```diff
@@ -3,7 +3,8 @@ import Select from "react-select";
 import { useCustomModal } from "../../../hooks/use-custom-modal";
 import { DrawingContentModelType } from "../../drawing-tool/model/drawing-content";
 import { VariableDrawingObjectData } from "../../drawing-tool/model/drawing-objects";
-import { findVariable, getVariables } from "./drawing-utils";
+import { Variable } from "@concord-consortium/diagram-view";
```

Comment:
Channeling Kirk here, this import should be moved above the others because it from a npm module instead of a local file.